ably/realtime_conn_spec_test.go

```diff
@@ -1078,3 +1078,64 @@ func TestRealtimeConn_RTN16(t *testing.T) {
 func sameConnection(a, b string) bool {
 	return strings.Split(a, "-")[0] == strings.Split(b, "-")[0]
 }
+
+type wrapProtoMessage struct {
```

Comment:
`wrapProtoMessage` doesn't really express what this does. 

* The wrapped thing is the `proto.Conn`, not the protocol message.
* The conn is wrapped to add a hook on incoming protocol messages.

Suggestion: How about `protoConnWithReceiveHook`?

Reply:
addressed d6515f72862ebc2d48deea576e03f249056e4c2b